### ISSUE TITLE
bug: Youtube Shorts Video not adjusting when NavBar removed

### ISSUE THREAD

**Issue by Stale-1** [revanced/revanced-patches#2326]
### Type

Cosmetic

### Bug description

After removing the NavBar in Shorts, the Video doesn't come down and occupy the whole screen leaving the Bottom part of the Screen with Black Strip.

### Steps to reproduce

1) Patch Youtube and make sure to include the options to hide Shorts Components.
2) Toggle the Hide NavBar in Shorts then restart.
3) Open Youtube again and  play shorts to see the black strip at the bottom of the screen.

### Relevant log output

```shell
Screenshot Provided Below.
```

### Screenshots or videos

![IMG_20230531_114512](https://github.com/revanced/revanced-patches/assets/74855954/ca35ef68-9c4f-4e1d-ab7a-35c2e181ab46)

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by n1cKz1** [comment]
I can reproduce this on my device by following these steps:
1. Open a shorts video
2. Click on comments section
3. Close comments section, navbar will immediately show up normally even if it's disabled
4. Scroll downwards (or upwards if there's a previous vid) to the next video, now there's an empty black space on the bottom just like in the screenshot

is it always like that for you?

**Comment by Stale-1** [comment]
> 

Yeah, I did encounter that. If I look in comments the NavBar will appear and if I scroll up/down it will disappear.

**Comment by n1cKz1** [comment]
> It now appears the nav bar shows up no matter what (hide nav is on, or off). At least, that's what I am seeing.

It's been like that since v2.187.0, at least for me

**Comment by RamPawar7** [comment]
IDK if anyone need this but I have accidentally found a workaround for this issue. Just tap the search icon in the shorts player to open search, then navigate back to the player. The bottom tab bar will be gone and the video will also adjusted to fit the screen.
